fastlane/Fastfile

```diff
@@ -748,7 +801,7 @@ platform :ios do
       type: 'enterprise',
       team_id: get_required_env('INT_EXPORT_TEAM_ID'),
       readonly: true,
-      app_identifier: ALL_BUNDLE_IDENTIFIERS.map { |id| id.sub('org.wordpress', 'org.wordpress.alpha') }
+      app_identifier: ALL_BUNDLE_IDENTIFIERS.map { |id| id.sub(APP_STORE_VERSION_BUNDLE_IDENTIFIER, 'org.wordpress.alpha') }
```

Comment:
@mokagio After a good night sleep I woke up this morning with the realization that it would have been a better idea to maybe have `ALL_BUNDLE_IDENTIFIERS` instead be a function dynamically generating the list of identifiers from a base one, and having that base one be the `APP_STORE_VERSION_BUNDLE_IDENTIFIER` by default.

e.g change [those lines 29-41 above](https://github.com/wordpress-mobile/WordPress-iOS/pull/16805/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R29-R41) to something like
```ruby
def all_bundle_identifiers(root: APP_STORE_VERSION_BUNDLE_IDENTIFIER)
  extensions_bundle_ids = %w[
    WordPressShare
    WordPressDraftAction
    WordPressTodayWidget
    WordPressStatsWidgets
    WordPressNotificationServiceExtension
    WordPressNotificationContentExtension
    WordPressAllTimeWidget
    WordPressThisWeekWidget
    WordPressIntents
  ].map { |suffix| "#{root}.#{suffix}" }

  [root, *extensions_bundle_ids]
end
```

And then in most places use `all_bundle_identifiers` (and let the default value for `root` be used), but in places like here for the alpha, use `all_bundle_identifiers(root: 'org.wordpress.alpha')` instead (which we could consider abstracting away into a `ALPHA_VERSION_BUNDLE_IDENTIFIER` constant at the top of the Fastfile, next to the AppStore one)

wdyt?

Reply:
Nice! Will open a PR to adopt this ✨